systems/data-plan/rate/pkg/db/markup_repo_test.go

```diff
@@ -569,7 +569,6 @@ func TestMarkupRepo_Update(t *testing.T) {
 }
```

Comment:
Good. We can also define the following in a `SetupTestDb()` function iin order to avoid duplication
```
                var err error

		db, mock, err := sqlmock.New() // mock sql.DB
		assert.NoError(t, err)

		// Simulate a database error
		mock.ExpectQuery(regexp.QuoteMeta(`SELECT`)).
			WillReturnError(errors.New("db error"))

		dialector := postgres.New(postgres.Config{
			DSN:                  "sqlmock_db_0",
			DriverName:           "postgres",
			Conn:                 db,
			PreferSimpleProtocol: true,
		})
		gdb, err := gorm.Open(dialector, &gorm.Config{})
```